### DISCUSSION TITLE
Welcome to building-secure-contracts Discussions!

### DISCUSSION THREAD

**Discussion by montyly** [crytic/building-secure-contracts#220] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ShabihEthSec** [comment]
Hello! I'm using VS Code and Foundry and am having trouble installing and setting up Echidna for testing in my workspace. Could anyone guide me on how to install and use it effectively? Thanks in advance!
